src/test/java/core/basesyntax/service/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ReaderServiceImplTest {
+    private static final String FILE_NOT_EXIST = "report";
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+    private ReaderService readerrService = new ReaderServiceImpl();

```

Comment:
```suggestion
    private ReaderService readerService = new ReaderServiceImpl();

```
readerService , you have to r letters

Reply:
done